sdk/forms/README.md

```diff
@@ -176,11 +176,11 @@ const form = createProtectedForm('#form-container', {
 
 #### Overview
 
-`createProtectedView` creates a secure view component to display sensitive data stored in Piiano Vault, ensuring that sensitive data is never directly exposed to the frontend.
+`createProtectedView` creates a secure view component to display sensitive data returned from Piiano Vault, ensuring that sensitive data is never directly exposed to the frontend.
```

Comment:
This sentence is confusing.  How would displaying sensitive data in the frontend ensures that sensitive data is never directly exposed to the frontend?

Reply:
It is also redundant with the "Goal".